docs/authentication/social-connections/atlassian.mdx

```diff
@@ -1,10 +1,15 @@
+---
+title: Atlassian
+description: Learn how to setup social connection with Atlassian.
+---
+
 # Atlassian
 
 How to set up social connection with Atlassian
 
 ## Overview
 
-Adding social connection with Atlassian to your app with Clerk is straightforward - you will need to populate the **Client ID**, **Client Secret** and **Authorized redirect URI** in your instance settings.
+Adding social connection with Atlassian to your app with Clerk is simple - you will need to populate the **Client ID**, **Client Secret** and **Authorized redirect URI** in your instance settings.

```

Comment:
I like to try and avoid words like "simple," "just," "only," or "easy." as it can trivialize the effort people need to put in and doesn't convey much meaning.

```suggestion
Adding social connection with Atlassian to your app with Clerk is done in a few  steps - you will need to populate the **Client ID**, **Client Secret** and **Authorized redirect URI** in your instance settings.

```